PR-Trigger/index.js

```diff
@@ -6,7 +6,6 @@ const {executePrActions} = require("../bin/execute-pr-actions")
 const {mergeCodeOwnersOnGreen} = require("../bin/side-effects/merge-codeowner-prs")
 const verify = require("@octokit/webhooks/verify");
 const sign = require("@octokit/webhooks/sign");
-const {runQueryToGetPRMetadataForStatus} = require("../bin/queries/status-to-PR-query")

```

Comment:
because a status was disconnected from a PR, you would use the GH API to search for a PR containing that commit - hacky, but worked, we don't need this now